addons/udes_stock/models/stock_picking.py

```diff
@@ -228,13 +228,25 @@ def get_move_lines(self, done=None):
                 move lines associated to picking
         :returns: Move lines of picking
         """
-        mls = self.move_line_ids
+        mls = self.get_fast_move_lines()
         if done:
-            return self.move_line_ids.get_lines_done()
+            return mls.get_lines_done()
         elif done == False:
-            return self.move_line_ids.get_lines_incomplete()
+            return mls.get_lines_incomplete()
         return mls
 
+    def get_fast_move_lines(self, aux_domain=None):
```

Comment:
Why aren't we using aux_domain on the `stock.move.line` function `get_fast_move_lines()` to pass in `[("picking_id", "in", self.ids)]` and

Can't we just do
```python
StockMoveLine = self.env["stock.move.line"]
mls = StockMoveLine.get_fast_move_lines(aux_domain=[("picking_id", "in", self.ids)])
```

Reply:
Needed the method in case that we need to override it , was Miquel's suggestion

Reply:
can we keep the method and replace with the above then?